### ISSUE TITLE
Scheduled snapshots are not getting created becasue of not being able to start etcd-rolling-snapshots container successfully.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#16511]
Rancher server version - v2.1.2-rc8

Steps to reproduce the problem:
Create a DO cluster.
By default , we have the snapshot enabled.

 etcd-rolling-snapshots containers fails to start successfully in etcd nodes:

```
3f28da5c47e9        rancher/coreos-etcd:v3.2.18                                                                                               "/usr/local/bin/et..."   10 minutes ago      Up 10 minutes                                    etcd
root@sangeetestsnap1:~# docker logs -f c241fe58
Incorrect Usage.

NAME:
   rke-etcd-backup rolling-backup - Perform rolling backups

USAGE:
   rke-etcd-backup rolling-backup [command options] [arguments...]

OPTIONS:
   --endpoints value  Etcd endpoints (default: "127.0.0.1:2379")
   --creation value   Create backups after this time interval in minutes (default: 5m0s)
   --retention value  Retain backups within this time interval in hours (default: 24h0m0s)
   --debug            Verbose logging information for debugging purposes [$RANCHER_DEBUG]
   --once             Take backup only once
   --name value       Backup name to take once
   --cacert value     Etcd CA client certificate path [$ETCD_CACERT]
   --cert value       Etcd client certificate path [$ETCD_CERT]
   --key value        Etcd client key path [$ETCD_KEY]
   
root@sangeetestsnap1:~# 
```

**Comment by alena1108** [comment]
Resolved in rancher/rancher:v2.1.2-rc9

**Comment by sangeethah** [comment]
Tested with rancher server version - v2.1.2-rc9
Create a DO cluster.
By default , we have the snapshot enabled with snapshot creation set to 12h and retention set to 72 h.

 etcd-rolling-snapshots container is able to start successfully
```
root@sangeenew1:~# docker logs -f etcd-rolling-snapshots
time="2018-11-09T16:33:07Z" level=info msg="Initializing Rolling Backups" creation=12h0m0s retention=72h0m0s 
```
